symtabAPI/src/emitElfStatic-ppc64.C

```diff
@@ -690,13 +695,12 @@ default:
                        static_cast<unsigned long>(targetData[dest+2]),
                        static_cast<unsigned long>(targetData[dest+3]));
 	}
-    if (branch_pred >= 0) {
-	unsigned int *td = (unsigned int *) targetData;
-	unsigned int target;
-	target = td[dest/4];
-	target = setBits(target, 10, 1, branch_pred);
-        memcpy(&td[dest/4], &target, sizeof(Elf32_Word));
-    } 
+  if (branch_pred >= 0) {
+    auto *td = &targetData[dest/4];
```

Comment:
targetData was of type char *.
The address of targetData[dest/4] is targetData + (dst/4) * sizeof (char);
In the origina code, td was a pointer of unsigned int *;
Won't address of  td[dest/4] be targetData + (dst/4) * sizeof(unsigned int)?

Reply:
Derp, yes. Good catch.